src/scene/scene_phongmaterial.js

```diff
@@ -522,31 +530,79 @@ pc.extend(pc, function () {
                     prefilteredCubeMap128.minFilter = pc.FILTER_LINEAR_MIPMAP_LINEAR;
                     prefilteredCubeMap128.magFilter = pc.FILTER_LINEAR;
                     prefilteredCubeMap128._prefilteredMips = true;
+                } else if (!allMips && !useTexCubeLod && prefilteredCubeMap128._levels.length>=6) {
+                    // Single -> multiple (provided single cubemap, but no texCubeLod)
+                    mips = null;
+                    if (pc.cubeMapMipsCache) {
+                        mips = pc.cubeMapMipsCache[prefilteredCubeMap128];

```

Comment:
There are various issues with this cache. First of all it's global which means we can't use it in the editor where we use the same engine for multiple contexts. So this cache needs to be tied to an instance of something... I have a feeling this belongs to the cubemap resource loader.. 

Second I don't think you can use an object as a hash key because objects are casted to strings. Which means pc.cubeMapMipsCache[new pc.Texture()] will return the same result as pc.cubeMapMipsCache[prefilteredCubeMap128].